Dockerfile.cpu

```diff
@@ -0,0 +1,61 @@
+FROM ubuntu:18.04 as build-stage
+
+WORKDIR /usr/src/app
+
+ENV LANG="C.UTF-8" LC_ALL="C.UTF-8" PATH="/opt/venv/bin:$PATH" PIP_NO_CACHE_DIR="false" DISPLAY=":0" CXXFLAGS="-I/opt/venv/lib/python3.6/site-packages/numpy/core/include"
```

Comment:
This sets the C++ include dir only; not sure it's what you want.

Maybe better to set / append to global env vars

    C_INCLUDE_DIR
    CXX_INCLUDE_DIR

and then also LD_LIBRARY_DIR

(if I remember correctly from C/CXX land)

Reply:
Nice  ! Trying this out now 👍

Reply:
> This sets the C++ include dir only;
Yes just need the numpy headers